package.json

```diff
@@ -323,9 +316,7 @@
     "pngjs": "^3.4.0",
     "polished": "^3.7.2",
     "prop-types": "^15.7.2",
-    "proper-lockfile": "^3.2.0",
```

Comment:
Looks like `@types/proper-lockfile` can also be removed

Reply:
Pushed with 8e0db00